src/init/init.ts

```diff
@@ -73,16 +73,15 @@ function checkIsMinGW(): boolean {
  * read the description from its package.json.
  */
 function getGeneratorDescription(generator: YeomanEnvironment.GeneratorMeta, generatorName: string): GeneratorDescription {
-  const name = getDisplayName(generatorName);
   let description: string = '';
+  let genName = generatorName;
 
-  if (templateDescriptions.hasOwnProperty(name)) {
-    description = templateDescriptions[name];
-  } else if (generator.resolved && generator.resolved !== 'unknown') {
+  if (generator.resolved && generator.resolved !== 'unknown') {
     try {
       const metapath = findup('package.json', {cwd: generator.resolved});
       const meta = JSON.parse(fs.readFileSync(metapath, 'utf8'));
-      description = meta.description;
+      description = meta.description || '';
```

Comment:
can we still default to description here instead of `''`? Like `description = meta.description || description`;